doc/kernel/other/ring_buffers.rst

```diff
@@ -74,7 +115,7 @@ Defining a Ring Buffer
 A ring buffer is defined using a variable of type :c:type:`struct ring_buf`.
 It must then be initialized by calling :cpp:func:`sys_ring_buf_init()`.
 
-The following code defines and initializes an empty ring buffer
+The following code defines and initializes an empty ring buffer for data items
```

Comment:
change to ``The following code defines and initializes an empty **data item mode** ring buffer``